src/ViewHoldingsRecord.js

```diff
@@ -82,6 +84,7 @@ class ViewHoldingsRecord extends React.Component {
     };
     this.craftLayerUrl = craftLayerUrl.bind(this);
     this.cViewMetaData = props.stripes.connect(ViewMetaData);
+    this.getPaneHeaderActionMenu = this.getPaneHeaderActionMenu.bind(this);
```

Comment:
One more `bind` we can avoid with a `() => {}`.

Reply:
Whoops. I'll update that ASAP.

Reply:
Updated.